File: C++ solutions/CodeForce/474a.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    char shift;
    string typed;
    cin >> shift >> typed;

    string keyboard = "qwertyuiopasdfghjkl;zxcvbnm,./";
    string result = "";

    for (char c : typed) {
        int pos = keyboard.find(c);
        if (shift == 'R') {
            result += keyboard[pos - 1]; 
        } else { 
            result += keyboard[pos + 1]; 
        }
    }

    cout << result << endl;
    return 0;
}

File: Python solutions/CodeForces/474a.py
def main():
    shift = input().strip()  
    typed = input().strip()

    keyboard = "qwertyuiopasdfghjkl;zxcvbnm,./"
    result = []

    for c in typed:
        pos = keyboard.index(c)
        if shift == 'R':
            result.append(keyboard[pos - 1])   
        else:  
            result.append(keyboard[pos + 1])  

    print("".join(result))

if __name__ == "__main__":
    main()
